inst/Upgrades.qml

```diff
@@ -37,14 +36,405 @@ Upgrades
+			{
+				switch(options["parallelAnalysisMethod"])
+				{
+					case "pc":		return "principalComponentBased";
+					case "fa":		return "factorBased";
+				}
+			}
+		}
+		ChangeRename
+		{
+			from:	"eigenValuesBox"
+			to:		"eigenValuesAbove"
+		}
+		ChangeRename
+		{
+			from:	"numberOfFactors"
+			to:		"manualNumberOfFactors"
+		}
+		ChangeRename
+		{
+			from:	"fitmethod"
+			to:		"factoringMethod"
+		}
+		ChangeRename
+		{
+			from:	"basedOn"
+			to:		"analysisBasedOn"
+		}
+		ChangeJS
+		{
+			name:		"analysisBasedOn"
+			jsFunction:	function(options)
+			{
+				switch(options["analysisBasedOn"])
+				{
+					case "correlation":		return "correlationMatrix";
+					case "covariance":		return "covarianceMatrix"
+				}
+			}
+		}
+		ChangeRename
+		{
+			from:	"highlightText"
+			to:		"loadingsDisplayLimit"
+		}
+		ChangeRename
+		{
+			from:	"factorLoadingsSort"
+			to:		"factorLoadingsOrder"
+		}
+		ChangeRename
+		{
+			from:	"incl_structure"
+			to:		"factorStructure"
+		}
+		ChangeRename
+		{
+			from:	"incl_correlations"
+			to:		"factorCorrelations"
+		}
+		ChangeRename
+		{
+			from:	"incl_fitIndices"
+			to:		"fitIndices"
+		}
+		ChangeRename
+		{
+			from:	"incl_pathDiagram"
+			to:		"pathDiagram"
+		}
+		ChangeRename
+		{
+			from:	"incl_screePlot"
+			to:		"screePlot"
+		}
+		ChangeRename
+		{
+			from:	"screeDispParallel"
+			to:		"screePlotParallelAnalysisResults"
+		}
+		ChangeRename
+		{
+			from:	"kmotest"
+			to:		"kaiserMeyerOlkinTest"
+		}
+		ChangeRename
+		{
+			from:	"bartest"
+			to:		"bartlettTest"
+		}
+		ChangeRename
+		{
+			from:	"missingValues"
+			to:		"naAction"
+		}
+		ChangeJS
+		{
+			name:		"factoringMethod"
+			jsFunction:	function(options)
+			{
+				switch(options["factoringMethod"])
+				{
+					case "minres":	return "minimumResidual";
+					case "ml":		return "maximumLikelihood";
+					case "pa":		return "principalAxis";
+					case "ols":		return "ordinaryLeastSquares";
+					case "wls":		return "weightedLeastSquares";
+					case "gls":		return "generalizedLeastSquares";
+					case "minchi":	return "minimumChiSquare";
+					case "minrank":	return "minimumRank";
+				}
+			}
+		}
+	}
+
+
+	Upgrade
+	{
+		functionName: 		"ConfirmatoryFactorAnalysis"
+		newFunctionName:	"confirmatoryFactorAnalysis"
 		fromVersion:		"0.16.3"
 		toVersion:			"0.16.4"
 
-		ChangeRename	{ from: "misfitplot";			to: "misfitPlot"			}
-		ChangeRename	{ from: "pathplot";				to: "pathPlot"				}
-		ChangeRename	{ from: "plotpars";				to: "pathPlotParameter"		}
-		ChangeRename	{ from: "plotstd";				to: "pathPlotStandardized"	}
-		ChangeRename	{ from: "plotmeans";			to: "pathPlotMean"			}
+		ChangeRename	{ from: "misfitplot";				to: "misfitPlot"				}
+		ChangeRename	{ from: "pathplot";					to: "pathPlot"					}
+		ChangeRename	{ from: "plotpars";					to: "pathPlotParameter"			}
+		ChangeRename	{ from: "plotstd";					to: "pathPlotStandardized"		}
+		ChangeRename	{ from: "plotmeans";				to: "pathPlotMean"				}
```

Comment:
So unfortunately as we did not manage to merge this before the release because of the failing tests, we need to update this slightly...

These options: `misfitPlot`, `pathPlot`, `pathPlotParameter`, `pathPlotStandardized`, and `pathPlotMean`, are already in the `0.16.4 release`, so what we need here is to keep these in here, and put all the rest of option renames you introduce in the PR in a separate 
```qml
Upgrade
{
	fromVersion:	"0.16.4"
	toVersion:	"0.17.0
}
```